schema_utils.py

```diff
@@ -265,3 +265,12 @@ def is_regex(obj):
     def matches_regex(obj, regex):
         """Ensures that `obj` (a string) matches the given regex."""
         raise NotImplementedError
+
+    @staticmethod
+    def is_valid_email_list(obj):
+        """Ensures that `obj` (a list of strings) is a valid list of emails"""
+        def is_valid_email(email):
+            return '@' in email and not email.startswith('@') and \
```

Comment:
I agree that we can use a better validator. Why don't you define a reasonable regex? Email validation is a solved problem; you should be able to find guidance online.

Reply:
done